lib/health_web/plugs/current_user.ex

```diff
@@ -0,0 +1,12 @@
+defmodule HealthWeb.Plugs.CurrentUser do
+  @moduledoc """
+    Adds current_user to assigns if user is logged in
+    (for use in templates)
+    """
+
+  def init(options), do: options
+
+  def call(conn, _opts) do
+    Plug.Conn.assign(conn, :current_user, Pow.Plug.current_user(conn))
+  end
+end
```

Comment:
It ain't much to looks at, but sure as hell beats calling `Pow.Plug.current_user(@conn)` in the template every time!

Reply:
Hey man, just searched for Pow users in github to see what people are building. `Pow.Plug.Session` already sets a `:current_user` assigns already so no need for the plug or to call `Pow.Plug.current_user/1`, @current_user is already available :)

It's documented in the readme here, but I should add it to the `current_user/1` docs as well: https://hexdocs.pm/pow/README.html#current-user-and-sign-out-link

Reply:
Awesome, thanks for checking this out!

I'll have to do a refactor :)